recipes/knoten/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "knoten" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/USGS-Astrogeology/knoten/archive/{{ version }}.tar.gz
+  sha256: d671f1c8c32107cf1ef1d2dadf236ba98f9f19dcfb744e302cb20860748164b0
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [ win or py2k ]
+  number: 0
+
+requirements:
+  build:
+  host:
+    - python >=3.7,<3.8
+    - setuptools 
+    - pip 
+  run:
+    - ale >=0.3.2
+    - python >=3.7,<3.8 
+    - csmapi >=1.0
+    - gdal >=3.0.0
+    - jinja2
+    - {{ pin_compatible('numpy') }}
+    - plio
+    - pyproj
+    - requests
+    - scipy
+    - shapely
+    - usgscsm >=1.3.1
+
+test:
+  imports:
+    - knoten
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/USGS-Astrogeology/knoten
+  summary: 'A library to leverage python wrapped Community Sensor Models (CSMs) for common spatial/sensor operations and testing.'
+  description: Automated control network generation.
+  doc_url: https://github.com/USGS-Astrogeology/knoten#readme
+  dev_url: https://github.com/USGS-Astrogeology/knoten
+  license: Unlicense
+  license_file: LICENSE.md
```

Comment:
Is this under a CC0-1.0 and should we specify that or does CC0-1.0 technically fall under an Unlicense?

Reply:
Yes, please use `CC0-1.0`.

Reply:
@ladoramkershner I think because knoten wasn't chosen for github dissemination it was never updated to CC0 and was still unlicense. I updated it a while back but forgot to update it here so thanks for pointing that out.